### DISCUSSION TITLE
Error: Invalid request: `version` should be >= 2.

### DISCUSSION THREAD

**Discussion by Henry-Nweze** [vercel/vercel#5617] [Help]
Am experiencing this Error: Invalid request: `version` should be >= 2. when am trying to import my project from github repo which is linked to vercel met with the Error:Invalid request: `version` should be >= 2.
Any idea on how to resolve this issue thanks

**Comment by paulogdm** [comment]
Please share your "vercel.json" here.

**Comment by GIDEON-BABALOLA** [comment]
{
    "version" : "",
    "builds" :[
        {
            "src": "server.js",
            "use": "@vercel/node"
        }
    ]
}
